pkgdb/include/flox/pkgdb/write.hh

```diff
@@ -23,8 +24,10 @@ namespace flox::pkgdb {
 /** @brief A set of arguments used by @a flox::pkgdb::PkgDb::scrape. */
 using Target = std::tuple<flox::AttrPath, flox::Cursor, row_id>;
 
-/** @brief A queue of @a flox::pkgdb::Target to be completed. */
-using Todos = std::queue<Target, std::list<Target>>;
+/** @brief A stack of @a flox::pkgdb::Target to be completed.
```

Comment:
```suggestion
/**
 * @brief A stack of @a flox::pkgdb::Target to be completed.
```
I think Doxygen gets upset if `/** <DOC><NL><DOC>*/` is split over multiple lines ( throws off the indent level detection if I remember correctly ).